src/posts/react-image-gallery.md

```diff
@@ -0,0 +1,211 @@
+---
+title: 'How to use React Image Gallery to build a beautiful and responsive image gallery'
+date: '2022-09-06'
+tags: ['javascript', 'react', 'image-gallery']
+thumbnail: ../images/thumbnails/image-gallery-icon.png
+author: 'Ajeet Chaulagain'
+---
+
+import { QuickInfo } from "../components/utilities/quick-info.tsx"
+
+Suppose you are creating apps or web pages. In that case, you might need an image gallery to share the multiple images on our webpage with the best user interactions possible for the targeted audience. However, creating an image library from scratch that provides a great user experience for to end user requires quite an effort. In this tutorial, we will learn how to use different* APIs of the react-image-gallery* npm package to create a _beautiful and elegant image gallery_ for your need with minimal effort.
```

Comment:
use AIDA method for a better start.https://www.siegemedia.com/creation/aida-model#:~:text=The%20AIDA%20model%20is%20a,to%20convert%20readers%20into%20customers.